modules/ROOT/pages/auradb/support.adoc

```diff
@@ -51,7 +51,7 @@ All users can access the https://status.neo4j.io/[Aura Status page] to check the
 
 == Response times and office hours
 
-Local business hours are 8:00am to 6:00pm, Monday through Friday, excluding major holidays as detailed below.
+Local business hours are 8:00 AM to 6:00 PM, Monday through Friday, excluding major holidays as detailed below.
```

Comment:
It may be debatable, but this format is suggested in the Google style guide

Reply:
👍